.github/scripts/pack-debian-apt.js

```diff
@@ -81,7 +81,8 @@ PATH=$PATH:$PWD/bin eval $(PATH=$PATH:$PWD/bin node -p "require('./package').scr
       await qq.chmod([workspace, 'usr/lib', config.dirname, 'bin', config.bin], 0o755);
       await qq.chmod([workspace, 'DEBIAN/postinst'], 0o755);
       await qq.x(`ln -s "../lib/${config.dirname}/bin/${config.bin}" "${workspace}/usr/bin/${pjson.oclif.bin}"`);
-      await qq.x(`dpkg --build "${workspace}" "${qq.join(dist, debArch(arch), `${versionedDebBase}.deb`)}"`);
+      // Use xz compression until Debian supports zstd (see issue #482)
+      await qq.x(`dpkg --build --compression=xz "${workspace}" "${qq.join(dist, debArch(arch), `${versionedDebBase}.deb`)}"`);
```

Comment:
`--compression ` does not seems to be a valid flag in dpkg, could you confirm that this command works.

Reply:
Thanks for taking a look! You're right, that flag isn't valid. I saw it on a man page somewhere, but I must have been looking at the wrong command. I apologize for that.

I just pushed a commit that should fix it. As I said in the OP I don't really have a good environment to test this, but my understanding from [here](https://man7.org/linux/man-pages/man1/dpkg-deb.1.html) and [here](https://stackoverflow.com/a/73529537/931860) is that this should do the trick.

Reply:
Getting error:
dpkg: error: unknown option -Z